plugins/secrets-manager-aws-secret-manager/src/static/startup/providers/secrets/secretsManagerFactory.ts

```diff
@@ -1,47 +1,53 @@
 import { ConfigService } from "@nestjs/config";
 import {
   GetSecretValueCommand,
-  SecretsManagerClient
-} from "@aws-sdk/client-secrets-manager"
-import { Logger } from "@nestjs/common"
+  SecretsManagerClient,
+} from "@aws-sdk/client-secrets-manager";
+import { Logger } from "@nestjs/common";
 import { Secrets } from "./secrets";
 
 export const secretsManagerFactory = {
   provide: "AWS_SECRETS_MANAGER",
   useFactory: async (configService: ConfigService) => {
-    const logger = new Logger()
+    const logger = new Logger();
 
     const client = new SecretsManagerClient({
-      region: configService.get("AWS_REGION")
-    })
+      region: configService.get("AWS_REGION"),
+    });
 
-    var secrets: Partial<Record<Secrets, unknown>> = {}
+    var secrets: Partial<Record<Secrets, unknown>> = {};
 
     for (const path of Object.values(Secrets)) {
-      const [secret_id, secret_name] = path.split(":")
+      const [secret_id, secret_name] = path.split(":");
 
       try {
-        const response = await client.send(new GetSecretValueCommand({ SecretId: secret_id }))
+        const response = await client.send(
+          new GetSecretValueCommand({ SecretId: secret_id }),
```

Comment:
the comma at the end of line 25 seems odd

Reply:
gone